chrome/nav.js

```diff
@@ -1141,6 +1152,56 @@ function updateRoutePlanner( data ) {
 	}
 }
 
+function highlightVisited( data ) {
+	let ukey = Universe.getServer ( doc ).substr( 0, 1 );
+
+    // initial setup
+    if ( !data[ ukey + 'visit' ] ) {
+        data[ ukey + 'visit' ] = {};
+    }
+    if( !config.displayVisitedDecay ) {
+        decayTime = 180000; //3 minutes
+    } else {
+        var decayTime = 1000*config.displayVisitedDecay;
+    }
+    // directly save that we are on this tile;
+    data[ ukey + 'visit' ][ userloc ] = Date.now();
```

Comment:
somehow, fails to capture when navving in OS, but hopefully the pilot understands that they have visited the tile they are navving on.